server/server.go

```diff
@@ -141,13 +141,15 @@ func (s *Server) ConnectionCount() int {
 func (s *Server) getToken() *Token {
 	start := time.Now()
 	tok := s.concurrentLimiter.Get()
+	metrics.TokenUsageGauge.Set(float64(s.concurrentLimiter.Allocated()) / float64(s.concurrentLimiter.Capacity()))
```

Comment:
Maybe better to report both `capacity` and `allocated` to give more information?

Reply:
OK. More comments are comming.